### DISCUSSION TITLE
Able to select "Start from beginning" after you select a session.

### DISCUSSION THREAD

**Discussion by AtlAntA118** [f1multiviewer/issue-tracker#22] [Feature requests] [answered]
Kinda just like the website/apps from FOM. I just got dropped into the F2 sprint race and it would be nice to get the option to start from the beginning when starting a session which is already in progress.

**Comment by Kareltje1980** [comment]
And maybe make an option to choose master feed, when you change that it offsets all the openened feeds to the same offset.

**Comment by f1multiviewer** [comment, marked as answer]
This should be working next release, but syncing during time-shifted viewing is more difficult, and will be added later (hopefully?)
